testdata/p4_16_errors/issue122.p4

```diff
@@ -0,0 +1,55 @@
+#include <core.p4>
+#include <v1model.p4>
+
+header ethernet_t {
+    bit<48> dst_addr;
+    bit<48> src_addr;
+    bit<16> eth_type;
+}
+
+struct Headers {
+    ethernet_t eth_hdr;
+}
+
+struct Meta {
+}
+
+bit<16> simple_action() {
```

Comment:
I realize the terminology used in the compiler sources might be different than in the spec, but isn't this a function, not an action, according to spec language?

Reply:
Yes that is a mistake that happened during the test-case reduction. Feel free to rename it.